src/vzlogger.cpp

```diff
@@ -397,8 +397,8 @@ int main(int argc, char *argv[]) {
 		return (0);
 	}
 
-	// only logging to file if daemonizing
-	options.logging(options.daemon());
+	// This defines whether the logging threads are started. We always want that, right?
+	options.logging(true);
```

Comment:
No. there are use-case where this is not wanted. E.g. if it's called not to go into daemon mode and just to return a single reading.
Honestly I don't know whether somebody is still using that but why remove it?